### PR TITLE
Update about.html for issue #20

### PR DESCRIPTION
**Pull request by MasonT3**
Add link to freecodecamp.org and remove references to Node and Angular.

### CODE REVIEW

File: docs/about.html [hunk 1]
```diff
@@ -53,8 +53,8 @@ <h1 class="blog-title"><a href="index.html">Free Code Camp - OKC</a></h1>
         <div id="main">
             <article class="post">
                 <div class="entry-content">
-                    <p>Free Code Camp Oklahoma City is the Oklahoma City branch of freecodecamp. <br><br>
-                     Free Code Camp is a completely no-cost way to learn how to program utilizing: HTML, CSS, JavaScript, NodeJS and Angular.<br><br>
+                    <p>Free Code Camp Oklahoma City is the Oklahoma City branch of <a href="https://www.freecodecamp.org/" target="_blank" title="freeCodeCamp" rel="external">freecodecamp</a>. <br><br>
```
Comment (CarsonMcKinstry) [inline docs/about.html]: Make sure capitalize Free Code Camp

**Review by CarsonMcKinstry — COMMENTED** [review]
Once you've got the link fixed, I'll merge this!

**Comment by MasonT3** [comment]
Capitalized the website url.

File: docs/about.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <meta property="og:title" content="Free Code Camp - OKC">
     <meta property="og:url" content="http://freecodecampokc.org/">
     <meta property="og:site_name" content="Free Code Camp - OKC">
-    <meta property="og:description" content="Free Code Camp - OKC is a group for those who want to learn to code in the Oklahoma City area.  We meet at StarSpace46 every Sunday.">
+    <meta property="og:description" content="Free Code Camp - OKC is a group for those who want to learn to code in the Oklahoma City area.  We meet at StarSpace46 every Sunday."
```
Comment (CarsonMcKinstry) [inline docs/about.html]: Forgot a closing bracket on the meta tag

File: docs/about.html [hunk 3]
```diff
@@ -53,8 +53,8 @@ <h1 class="blog-title"><a href="index.html">Free Code Camp - OKC</a></h1>
         <div id="main">
             <article class="post">
                 <div class="entry-content">
-                    <p>Free Code Camp Oklahoma City is the Oklahoma City branch of freecodecamp. <br><br>
-                     Free Code Camp is a completely no-cost way to learn how to program utilizing: HTML, CSS, JavaScript, NodeJS and Angular.<br><br>
+                    <p>Free Code Camp Oklahoma City is the Oklahoma City branch of <a href="https://www.FreeCodeCamp.org/" target="_blank" title="FreeCodeCamp" rel="external">freecodecamp</a>. <br><br>
```
Comment (CarsonMcKinstry) [inline docs/about.html]: I meant the text inside of the anchor tag

**Review by CarsonMcKinstry — CHANGES_REQUESTED** [review]

**Comment by MasonT3** [comment]
Sorry, I got confused. I fixed it now.

**Comment by CarsonMcKinstry** [comment]
@MasonT3 Make sure to fix the other issue in `docs/about.html`. It looks like you're still missing a closing bracket on line 12.

**Comment by MasonT3** [comment]
Committed the closing bracket. Sorry this has been taking long. First time trying to figure out and contribute to open source

**Review by CarsonMcKinstry — APPROVED** [review]

**Comment by CarsonMcKinstry** [comment]
@MasonT3 Not a problem! One tip is that when you get a change request, go through each change and make sure they're at least updated before pushing again.

**Comment by MasonT3** [comment]
Thanks for the tip!

**Pull request merged**